.github/workflows/build.yml

```diff
@@ -110,12 +110,19 @@ jobs:
           artifacts/*.so
           artifacts/msh3app
           artifacts/msh3test
+          artifacts/h3ping
     - name: msh3app
       run: |
         ./artifacts/msh3app outlook.office.com
         ./artifacts/msh3app www.cloudflare.com
         ./artifacts/msh3app www.google.com
       timeout-minutes: 1
+    - name: h3ping
+      run: |
```

Comment:
I don't want the job to fail if h3ping returns any errors (i.e. on timeout), so please update it accordingly.